src/admin/upload_index.rs

```diff
@@ -33,23 +32,18 @@ pub fn run(opts: Opts) -> anyhow::Result<()> {
         return Ok(());
     }
 
-    let mut progress_update_time = Instant::now();
-    for (i, file) in files.iter().enumerate() {
+    let pb = ProgressBar::new(files.len() as u64);
+    pb.set_style(ProgressStyle::default_bar().template("{bar:60} ({pos}/{len}, ETA {eta})"));
+
+    for file in files.iter().progress_with(pb) {
         let crate_name = file.file_name().unwrap().to_str().unwrap();
         let path = repo.index_file(crate_name);
         if !path.exists() {
-            println!("skipping file `{}`", crate_name);
```

Comment:
Why remove this?

Reply:
it looks like when we call `println!()` while a progressbar is active it does not print the message, but instead freezes the progressbar and continues it on the next line. I guess it doesn't play well with the clearing of the current line.